src/browser/ui/dom/SVGDOMPropertyConfig.js

```diff
@@ -83,7 +114,18 @@ var SVGDOMPropertyConfig = {
     strokeOpacity: 'stroke-opacity',
     strokeWidth: 'stroke-width',
     textAnchor: 'text-anchor',
-    viewBox: 'viewBox'
+    viewBox: 'viewBox',
+    xlinkActuate: 'xlink:actuate',
```

Comment:
Luckily all the current browsers seems to ignore namespace set on the 2nd argument of setAttributeNS:

Testing it with the 'whatever' namespace.
http://jsfiddle.net/framp/yst5c721/1/
http://imgur.com/vXsHlGL